####-sparse-observable.md

```diff
@@ -0,0 +1,518 @@
+
+  Currently, we use `Pauli`/`PauliList` for this; our *abstract observables* are already represented in terms of the measurement terms (plus extra explicit identities).
+  The measurement terms are not required to be the same size as the abstract terms, even though our current system has this.
+
+### Considered use cases
+
+There are several use-cases for observables throughout the entire life cycle of "a user calls a primitive and gets a result":
+
+1. A user must be able to efficiently construct their operators, which may involve mathematical manipulations as they build up their problem.
+   This prioritises efficient construction and manipulation of the objects, while maintaining a clear mathematical meaning in the context of observables.
+
+2. All observables for an estimator pub / batch of pubs must exist simultaneously in memory on the client side and be transmitted over the wire, so must not use excessive memory.
+
+3. An estimator primitive (or circuit-knitting toolbox, etc) must be able to efficiently extract all the terms that correspond to unique measurements that must be made, across the entire broadcast `EstimatorPub`.
+   This is not necessarily the same set of mathematical terms as appear in a single operator, and the available groupings are dictated not just be individual mathematical operators or the `ObservablesArray`, but also in how it broadcasts against the `BindingsArray`.
+
+4. After the measurements are made in hardware execution, the values of the observable expectations must be recalculated after.
+   This means that the abstract observable should be able to easily and efficiently map back to the measurement terms needed for each of the individual abstract terms.
+
+Cases 1 and 2 are more about the user experience and input format of the primitives.
+The obvious difference between the desired alphabet of abstract terms there, and the more restricted set of measurement terms that estimator primitives need to work with internally suggests that case 3 would be better handled by a separate object.
+The operations that must be efficient for use-case 3 are also quite different to use-cases 1 and 2; in particular, slicing along qubitwise axes should be efficient for use-case 3, but there is no need to store the term weights or the sum structure in this backend intermediate representation.
+
+
+### `SparseObservable`: representation of observables
+
+> [!NOTE]
+> **Summary**: A new observable class in `quantum_info` will represent Paulis and eigenstate-projectors, and store only non-identity terms.
+> This will be for users' benefit, and will be the "natural" input format of `Estimator`.
+
+A new class, `qiskit.quantum_info.SparseObservable`, will (in the future) replace/supplement `SparsePauliOp` as the "abstract observable" input to `Estimator`.
+This will for construction, representation and transmission of one set of mathematical operations.
```

Comment:
will be used for?

Reply:
I honestly have no idea what's going on here.  It feels like three completely disjoint thoughts - maybe I got distracted while rewriting it or something.  I deleted it - whatever I was trying to say should be covered by the rest of the section anyway.